examples/hello-world/index.html

```diff
@@ -28,12 +28,12 @@
       .svg(d3.select("#tree_display"));
```

Comment:
It might be nice to have bl.ocks style pages. Most of the time when viewing examples I want to be able to view and/or copy the code quickly.